WasatchNET/Spectrometer.cs

```diff
@@ -3003,6 +3176,44 @@ protected void correctBadPixels(ref double[] spectrum)
             }
         }
 
+        protected void correctBin2x2(ref double[] spectrum)
+        {
+            if (eeprom.featureMask.bin2x2 && !areaScanEnabled)
+            {
+                var smoothed = new double[spectrum.Length];
+                if (eeprom.horizontalBinningMethod == EEPROM.HORIZONTAL_BINNING_METHOD.BIN_2X2)
+                {
+                    for (int i = 0; i < spectrum.Length - 1; i++)
+                        smoothed[i] = (spectrum[i] + spectrum[i + 1]) / 2.0;
+                    smoothed[spectrum.Length - 1] = spectrum[spectrum.Length - 1];
+                    spectrum = smoothed;
+                }
+                else if (eeprom.horizontalBinningMethod == EEPROM.HORIZONTAL_BINNING_METHOD.BIN_4X2_AVG)

```

Comment:
Oh, okay, we're calling all horizontal binning "bin2x2" in here. That addresses some of my comments in other PRs about non-2x2 options.